build/yamls/flow-visibility.yml

```diff
@@ -152,6 +152,40 @@ data:
     ADD COLUMN clusterUUID String;
     ALTER TABLE flows_local
     ADD COLUMN clusterUUID String;
+  000003_0-3-0.down.sql: |
```

Comment:
Is this a copy/paste of the migrations scripts, or do we have some generation code that automatically populates flow-visibility.yml?

(I think I might have already asked this question - in that case please accept my apologies)

Reply:
`flow-visibility.yml` is generated by running `make manifest`